src/System Application/App/AI/src/Copilot/CopilotCapabilityImpl.Codeunit.al

```diff
@@ -272,22 +272,12 @@ codeunit 7774 "Copilot Capability Impl"
     end;
 
     [EventSubscriber(ObjectType::Codeunit, Codeunit::"System Action Triggers", 'GetCopilotCapabilityStatus', '', false, false)]
-    local procedure GetCopilotCapabilityStatus(Capability: Integer; var IsEnabled: Boolean; AppId: Guid)
+    local procedure GetCopilotCapabilityStatus(Capability: Integer; var IsEnabled: Boolean; AppId: Guid; Silent: Boolean)
     var
         AzureOpenAI: Codeunit "Azure OpenAI";
-        CurrentModuleInfo: ModuleInfo;
         CopilotCapability: Enum "Copilot Capability";
-        Silent: Boolean;
     begin
-        NavApp.GetCallerModuleInfo(CurrentModuleInfo);
-        if AppId <> CurrentModuleInfo.Id() then
-            exit;
-
         CopilotCapability := Enum::"Copilot Capability".FromInteger(Capability);
-
-        if CopilotCapability = Enum::"Copilot Capability"::Chat then
-            Silent := true;
-
         IsEnabled := AzureOpenAI.IsEnabled(CopilotCapability, Silent);
```

Comment:
We need the app id:

IsEnabled := AzureOpenAI.IsEnabled(CopilotCapability, Silent, AppId);